src/Admin.php

```diff
@@ -209,6 +212,20 @@ public static function get_sample_permalink_html($html, $post_id, $new_title, $n
   }
 
   /**
+   * Prevents TinyMCE editor setting unwanted rel attribute values for links.
+   *
+   * TinyMCE sets attribute rel="noopener noreferrer" for links opening in new
+   * window (target="_blank"). This would prevent affiliates from correctly
+   * tracking those links.
```

Comment:
```diff
-This would prevent affiliates from correctly tracking those links.
+This can prevent affiliate marketing tools from correctly tracking such links (e.g. in sponsored posts), potentially causing less revenue, so we disable the behavior by default.
```